cmd/revad/config/config.go

```diff
@@ -52,7 +53,11 @@ func reGet(prefix string, kv *map[string]interface{}) {
 		if c, ok := val.(map[string]interface{}); ok {
 			reGet(prefix+"."+k, &c)
 		} else {
-			(*kv)[k] = v.Get(prefix + "." + k)
+			newVal := v.Get(prefix + "." + k)
+			fmt.Println("reGet", prefix, k, "val:", val, "->", newVal)
+			if newVal != nil {
```

Comment:
only overwrite config if newval is present, fixes using environment variables configuration